front/src/config/i18n/en.json

```diff
@@ -1556,36 +1556,76 @@
   "deviceFeatureUnit": {
```

Comment:
In this file, the `deviceFeatureCategory` attribute should be filled for new category

Reply:
https://github.com/GladysAssistant/Gladys/blob/master/front/src/config/i18n/en.json#L1608